packages/hoppscotch-backend/prisma/schema.prisma

```diff
@@ -68,9 +68,11 @@ model TeamRequest {
 }
 
 model Shortcode {
-  id         String   @id
+  id         String   @id @unique
   request    Json
+  properties Json? // if properties is null, SharedRequest is an shortcode else embed
```

Comment:
if this will be always embeds then why name it properties instead of embedsInfo or something like that

Reply:
I gave it a generic name because while the property may belong to that of an embed in the DB layer they are all stored in the same table, hence I gave it a generic name that conveys the meaning.

Reply:
so embeds are a property of shortcodes and not the other way around ?